slip-0024.md

```diff
@@ -188,12 +210,15 @@ k = Key(m/"SLIP-0024"/"Address MAC key")
 The address authentication code is computed as:
 
 ```
-mac = HMAC-SHA256(key = k, msg = coinType || address)
+mac = HMAC-SHA256(key = k, msg = addressInfo)
 ```
 
-where `coinType || address` is the concatenation of the following fields:
+where `addressInfo` is the concatenation of the following fields:
 
 * *coinType* (4 bytes): 32-bit encoding of the SLIP-0044 coin type of the address in little-endian byte order.
+* The BIP-0032 path to derive the address from the master node, which consists of:
+    * *n* (CompactSize integer): the path length.
+    * *i*<sub>1</sub> || *i*<sub>2</sub> || ... || *i*<sub>*n*</sub> (variable length): concatenation of the 32-bit encodings of the child indices, each in little-endian byte order.
```

Comment:
i don't fully understand the role of adding this to the `addressInfo`

if i understand the address verification mechanism, the wallet may not even have the derivation path available at the time of processing the payment request? the host would have to tell it, along with the address authentication code.

what sort of distinguishing feature are we adding here?

Reply:
> the host would have to tell it, along with the address authentication code.

Correct, and the host should have this information readily available.

I think this feature originates from a discussion between @tomasklim and me. The reason is that we may want to show the user the path to the address when they click on details in Trezor.